Doc/whatsnew/3.7.rst

```diff
@@ -865,143 +1667,245 @@ argument to generate a compressed archive.  A command line option
 (Contributed by Zhiming Wang in :issue:`31638`.)
 
 
+zipfile
+-------
+
+:class:`~zipfile.ZipFile` now accepts the new *compresslevel* parameter to
+control the compression level.
+(Contributed by Stephen J. Turnbull. in :issue:`28102`.)
+
+Subdirectories in archives created by ``ZipFile`` are now stored in
+alphabetical order.
+(Contributed by Bernhard M. Wiedemann in :issue:`30693`.)
+
+
+C API Changes
+=============
+
```

Comment:
Need to mention PEP 567 related changes to the C API: https://www.python.org/dev/peps/pep-0567/#c-api

Reply:
These C APIs seem to be undocumented, BTW.